ee/modules/110-istio/images/api-proxy/werf.inc.yaml

```
@@ -0,0 +1,50 @@
+---
+image: {{ .ModuleName }}/{{ .ImageName }}
+fromImage: common/distroless
+import:
+- image: {{ .ModuleName }}/{{ .ImageName }}-build-artifact
+  add: /src/api-proxy
+  to: /api-proxy
+  after: setup
+docker:
+  ENTRYPOINT: ["/api-proxy"]
+---
+image: {{ .ModuleName }}/{{ .ImageName }}-build-artifact
+from: {{ .Images.BASE_GOLANG_23_ALPINE }}
+final: false
+import:
+- image: {{ .ModuleName }}/{{ .ImageName }}-src-artifact
+  add: /src
+  to: /src
+  before: install
+mount:
+- fromPath: ~/go-pkg-cache
+  to: /go/pkg
+shell:
+  beforeInstall:
+  {{- include "alpine packages proxy" . | nindent 2 }}
+  - apk add --no-cache bash git binutils
+  install:
+  - cd /src
+  - go mod download
+  setup:
+  - cd /src
+  - export GOOS=linux GOARCH=amd64 CGO_ENABLED=0
+  - go build -ldflags="-s -w" -o /src/api-proxy main.go
+  - chmod 0500 /src/api-proxy
+---
+image: {{ .ModuleName }}/{{ .ImageName }}-src-artifact
+fromImage: common/src-artifact
+final: false
+git:
+- add: /{{ $.ModulePath }}modules/{{ $.ModulePriority }}-{{ $.ModuleName }}/images/{{ $.ImageName }}/src
+  to: /src
+  excludePaths:
+  - "*.md"
+  - "*.yaml"
+  stageDependencies:
+    install:
+    - go.mod
+    - go.sum
+    setup:
+    - "*.go"
```

shell is required otherwise werf will not handle updates to src correctly.
```suggestion
    - "*.go"
shell:
  install:
  - cd /src
```

---

fixed